src/mempool/own_lane.rs

```diff
@@ -92,9 +92,13 @@ impl super::Mempool {
             .lanes
             .get_lane_pending_entries(&self.own_lane_id(), last_cut)?;
 
+        let min_signatures = match self.staking.is_bonded(self.crypto.validator_pubkey()) {
+            true => 2,
```

Comment:
why 2 ?

Reply:
Maybe the name of the variable isn't great, but the logic below checks if we need to disseminate to all staked validators. To do this, it checks the numbers of signatures of the lane entry (which is always at least 1 - our own). Then two cases:
- we're just a lane manager - then we need to disseminate anyways
- we're a bonded validator - then there needs to be at least one other bonded validator or there's no point disseminating, so '2'.